src/meshes/p4est_mesh.jl

```diff
@@ -268,35 +300,118 @@ is non-trivial.
 - `unsaved_changes::Bool`: if set to `true`, the mesh will be saved to a mesh file.
 """
 function P4estMesh{NDIMS}(meshfile::String;
-                   mapping=nothing, polydeg=1, RealT=Float64,
-                   initial_refinement_level=0, unsaved_changes=true) where NDIMS
-  # Prevent p4est from crashing Julia if the file doesn't exist
+                          mapping=nothing, polydeg=1, RealT=Float64,
+                          initial_refinement_level=0, unsaved_changes=true) where NDIMS
```

Comment:
Would it make sense to add a kwarg `force_mapping=false` to allow using a HOHQMesh-enhanced Abaqus file and then still use the mapping provided here? E.g., for testing purposes?

Reply:
...maybe? I still cannot think of a good use case where we would take a HOHQMesh and then map it with a different function to approximate something.

Reply:
Well, I thought it might be useful sometimes to see, e.g., a straight-sided version without curvature in case the resulting mesh looks wrong. In this case, one could check if at least the connectivity is correct by forcing it to be linear elements.

But this was merely meant as a suggestion - if you do not see a benefit yourself, feel free to resolve this conversation without further action.

Reply:
How would you get a straight-sided version of a curved mesh (generated by HOHQMesh) by applying some mapping to it?

Reply:
Well, if you read in the Abaqus file and ignore the high-order information. But as I said, if you don't see a good reason to do it, let's follow YAGNI and just leave it as it is.

Reply:
> How would you get a straight-sided version of a curved mesh (generated by HOHQMesh) by applying some mapping to it?

You can still pass the HOHQMesh Abaqus file to the original p4est constructor. If you use the defaults `polydeg=1` and `mapping=nothing` it would create a straight-sided mesh from the corner list and element connectivity (all the commented boundary information is completely ignored by p4est).

Reply:
Ah ok, then this is still possible and should work as I intended! Thanks for the clarification!